tools/cli/src/test/java/org/apache/carbondata/tool/CarbonCliTest.java

```diff
@@ -260,6 +264,31 @@ public void testSummaryPageMeta() {
     Assert.assertTrue(output.contains(expectedOutput));
   }
 
+  @Test
+  public void testSummaryAllColumns() {
+    String[] args = { "-cmd", "summary", "-p", path, "-C" };
+    ByteArrayOutputStream out = new ByteArrayOutputStream();
+    PrintStream stream = new PrintStream(out);
+    CarbonCli.run(args, stream);
+    String output = new String(out.toByteArray());
+    System.out.println(output);
+    Assert.assertTrue(output.contains("Block  Blocklet  Column Name  Meta Size  Data Size"));
+  }
+
+  @Test
+  public void testSummaryAllColumnsForOneFile() {
+    CarbonFile folder = FileFactory.getCarbonFile(path);
+    CarbonFile[] carbonFiles =
+        folder.listFiles(file -> file.getName().endsWith(CarbonTablePath.CARBON_DATA_EXT));
+    String[] args = { "-cmd", "summary", "-p", carbonFiles[0].getCanonicalPath(), "-C" };
+    ByteArrayOutputStream out = new ByteArrayOutputStream();
+    PrintStream stream = new PrintStream(out);
+    CarbonCli.run(args, stream);
+    String output = new String(out.toByteArray());
+    System.out.println(output);
```

Comment:
avoid directly printing to console, use logger instead

Reply:
removed